scripts/mkcloud

```diff
@@ -479,6 +479,14 @@ function crowbar_register
     done
 }
 
+# setup an NFS server on the first lonely node
+function lonelynode_nfs_server
+{
+    local nfsserver=($(nodes ids lonely))
+    local mac=$(macfunc $nfsserver)
```

Comment:
then lets keep it for now - I am reworking the nodes function anyway and will have to adapt a lot then :)